include/picongpu/param/incidentField.param

```diff
@@ -0,0 +1,114 @@
+/* Copyright 2020-2021 Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file incidentField.param
+ *
+ * Load incident field parameters
+ */
+
+#pragma once
+
+#include "picongpu/fields/incidentField/Profiles.def"
+
+
+namespace picongpu
+{
+    namespace fields
+    {
+        namespace incidentField
+        {
+            /** Functor to set values of incident E field
+             */
+            class FunctorIncidentE
+            {
+            public:
+                /* We use this to calculate your SI input back to our unit system */
+                PMACC_ALIGN(m_unitField, const float3_64);
+
+                HDINLINE FunctorIncidentE(const float3_64 unitField) : m_unitField(unitField)
+                {
+                }
+
+                /** Calculate incident field E_inc(r, t) for a source
+                 *
+                 * \param cellIdx The total cell id counted from 0, note that it is fractional
+                 * \param currentStep The current time step, note that it is fractional
+                 */
+                HDINLINE float3_X operator()(const floatD_X& cellIdx, const float_X currentStep) const
+                {
+                    auto const fieldSI = float3_X(0.0_X, 0.0_X, 0.0_X);
+                    return fieldSI / precisionCast<float_X>(m_unitField);
+                }
+            };
+
+            /** Functor to set values of incident B field
+             */
+            class FunctorIncidentB
+            {
+            public:
+                /* We use this to calculate your SI input back to our unit system */
+                PMACC_ALIGN(m_unitField, const float3_64);
+
+                HDINLINE FunctorIncidentB(const float3_64 unitField) : m_unitField(unitField)
+                {
+                }
+
+                /** Calculate incident field B_inc(r, t) for a source
+                 *
+                 * \param cellIdx The total cell id counted from 0, note that it is fractional

```

Comment:
Could you please grep in your changes and change `\param` into `@param` and `\result` into `@result` see: https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#5-doxygen-comments

Reply:
Done